ginga/util/io_fits.py

```diff
@@ -157,19 +156,18 @@ def load_hdu(self, hdu, dstobj=None, **kwargs):
 
         return dstobj
 
-
     def load_file(self, filespec, numhdu=None, dstobj=None, memmap=None,
                   **kwargs):
-
+        inherit_primary_header = kwargs.pop('inherit_primary_header', False)
```

Comment:
This is the fix for "inherit_primary_header" bug. It is unrelated to `LineProfile` but I throw it in here because I need this fix for the test image to work because it has messed up headers.